das/aggregator.go

```diff
@@ -312,11 +312,14 @@ func (a *Aggregator) Store(ctx context.Context, message []byte, timeout uint64,
 	aggCert.Sig = blsSignatures.AggregateSignatures(sigs)
 	aggPubKey := blsSignatures.AggregatePublicKeys(pubKeys)
 	aggCert.SignersMask = aggSignersMask
-	copy(aggCert.DataHash[:], expectedHash)
+	aggCert.DataHash = expectedHash
 	aggCert.Timeout = timeout
 	aggCert.KeysetHash = a.keysetHash
 
-	verified, err := blsSignatures.VerifySignature(aggCert.Sig, serializeSignableFields(&aggCert), aggPubKey)
+	// Rachel TODO: count up requiredServicesForStore number on each version
```

Comment:
Please check: Does this TODO need to be done, or should it be deleted, or should it stay in the code?

Reply:
I've removed it -- since we're doing an upgrade on all the DASes this doesn't require synchronization